templates/rhdm76-authoring-ha.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     openshift.io/display-name: Red Hat Decision Manager 7.6 authoring environment (HA, persistent, with https)
     openshift.io/provider-display-name: Red Hat, Inc.
     description: Application template for a HA persistent authoring environment, for Red Hat Decision Manager 7.6 - Deprecated
-    template.openshift.io/long-description: This template defines resources needed to develop and execute Rules on Red Hat Decision Manager 7.6 with High Availability enabled as tech preview, including application deployment configuration, secure and insecure http communication and persistent volumes to store data that needs to survive a restart. Deprecated in 7.6; using the Red Hat Business Automation Operator is recommended.
+    template.openshift.io/long-description: This template defines resources needed to develop and execute Rules on Red Hat Decision Manager 7.6 with High Availability enabled as tech preview, including application deployment configuration, secure and insecure http communication and persistent volumes to store data that needs to survive a restart. Template for Red Hat OpenShift Container Platform version 3.11. Deprecated since Red Hat Decision Manager version 7.5; consider using the Red Hat Business Automation Operator.
```

Comment:
there is a 7.5 reference, update to 7.6

Reply:
@spolti all these 7.5 references are intentional. They inform the reader that templates have been deprecated SINCE version 7.5. This remains true for subsequent version 7.6.